### PR TITLE
#8 task.andrei.manya

### PR DESCRIPTION
**Pull request by AndreiManya**

### CODE REVIEW

File: task/08-objects-tasks.js [hunk 1]
```diff
@@ -23,7 +23,24 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+//   let r ={};
+//   r.width = width;
+//   r.height = height;
```
Comment (AlexMokin) [inline task/08-objects-tasks.js]: remove commented code

**Review by AlexMokin — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexMokin** [comment]
please add travis ci link

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AndreiManya** [comment]
https://travis-ci.org/github/AndreiManya/js-assignments/builds/709447097

**Comment by AlexMokin** [comment]
accepted

**Pull request closed without merging**